tabs/src/services/lnbitsServiceLocal.ts

```diff
@@ -740,6 +763,68 @@ const getNostrRewards = async (adminKey: string, stallId: string): Promise<Nostr
   }
 };
 
+interface Transaction {
+  checking_id: string;
+  pending: boolean;
+  amount: number; // in millisatoshis (msats)
+  fee: number;
+  memo: string;
+  time: number; // Unix timestamp
+}
+
+interface WalletTransaction {
```

Comment:
What is the difference between `Transaction` and `WalletTransaction`?

Reply:
The main difference between Transaction and WalletTransaction is how they structure the data and their purposes within the context:

1. Transaction:
This interface represents a single transaction in a wallet.
Each Transaction contains details about a specific transaction such as the amount, fee, memo, timestamp (time), and whether it's pending.

**interface Transaction {
  checking_id: string;
  pending: boolean;
  amount: number;  // in millisatoshis (msats)
  fee: number;
  memo: string;
  time: number;  // Unix timestamp
}**

A Transaction represents a single record of money being sent or received in a wallet.

2. WalletTransaction:
This interface is a container for multiple transactions belonging to a specific wallet.
It groups multiple Transaction objects under a wallet_id (which identifies the wallet they belong to).

**interface WalletTransaction {
  wallet_id: string;
  transactions: Transaction[];
}**
A WalletTransaction groups all the Transaction records for a specific wallet and allows you to associate a list of transactions with a particular wallet.

I would like to leave it as is